packages/ibm-products/src/components/ConditionBuilder/ConditionBuilder.types.ts

```diff
@@ -5,7 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 import { TextAreaProps, TextInputProps } from '@carbon/react';
-import { CarbonIconType } from '@carbon/react/icons';
+import { CarbonIconType } from '@carbon/icons-react';
```

Comment:
Same thing here

Reply:
![Screenshot 2025-01-10 at 11 55 58 AM](https://github.com/user-attachments/assets/6d5380fb-2381-49b0-9696-d81996fb6af4)

I changed it for this reason

Reply:
So i had to change the import to fix this error